meta/main.yml

```diff
@@ -15,7 +15,7 @@ galaxy_info:
   # - GPLv3
   # - Apache
   # - CC-BY
-  license: license (GPLv2, CC-BY, etc)
+  license: LGPLv2
```

Comment:
Marking for reference regarding license question posed in another comment.

Reply:
Changed to LGPLv3